packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -90,7 +91,11 @@ export class MonthPicker extends React.PureComponent {
       const selectedMonths = monthNumbers.filter(
         m => !disabledMonths.includes(m)
       );
-      this.setState({ selectedMonths });
+      this.setState({
+        selectedMonths,
+        selectAllPressed: true,
+        clearAllPressed: false
```

Comment:
> Forgot to revert these

@1Copenut @pwolfert Bringing the convo down to this (uncompiled) file. Are these two lines (`selectAllPressed`, `clearAllPressed`) supposed to still be here or no?

Reply:
No, they should not. I missed those on code cleanup, and will remove them now. Thanks all.